src/zope/app/applicationcontrol/browser/runtimeinfo.py

```diff
@@ -12,15 +12,17 @@
 #
 ##############################################################################
 """Define runtime information view component for Application Control
-
-$Id$
 """
 __docformat__ = 'restructuredtext'
 
 from zope.app.applicationcontrol.interfaces import IRuntimeInfo
 
 from zope.app.applicationcontrol.i18n import ZopeMessageFactory as _
 
+try:
+    long
+except NameError:
+    long = int
```

Comment:
Oh, cool, I didn't realize Python 2 would handle the overflow automatically. TIL!

```pycon
>>> sys.maxint + 1
9223372036854775808L
>>> i = 9223372036854775808L
>>> int(i)
9223372036854775808L
```